src/util/cmdline.h

```diff
@@ -76,6 +76,9 @@ class cmdlinet
   std::string get_value(char option) const;
   std::string get_value(const char *option) const;
 
+  std::optional<std::string> value_opt(char option) const;
+  std::optional<std::string> value_opt(const char *option) const;
+
```

Comment:
How about changing this API to become `value_or`? So that it would be used like this: `cmdline.value_or("log", "-")`?

Reply:
Might be confusing given that it is the same method name as `std::optional::value_or`. Right now what you do is only a little bit longer: `cmdline.value_opt("log").value_or("-")`.

Reply:
We could consider `operator()`? Then it would be `cmdline("log").value_or("-")`

Reply:
Perhaps `value_or` isn't the best name to choose, but it still seems like a nuisance when the only thing we really want is `value_opt` immediately followed by `value_or`.

Reply:
There is some (very modest) benefit in doing
`auto foo_opt = cmdline.value_opt("foo");
if(foo_opt) ....  foo_opt.value()`

The benefit is that you avoid the double lookup in the argument map that you'd do with `cmdline.isset("foo")` followed by `cmdline.get_value()`. But of course not measurable.